engine/labels/labels_test.go

```diff
@@ -159,6 +159,12 @@ func TestParseLabel(t *testing.T) {
 			wantKey: "konveyor.io/fact",
 			wantVal: "Spring Beans",
 		},
+		{
+			name:    "version ranges in values",
```

Comment:
should we add tests for the `+`, `-` variations as well?

Reply:
@fabianvf added